README.md

```diff
@@ -14,26 +14,24 @@ A [Singer](https://singer.io/) postgres target, for use with Singer streams gene
 
 ## Usage
 
-Create a config file with postgres connection information and target postgres schema.
-
-```json
-{
-  "postgres_host": "locahost",
-  "postgres_port": 5432,
-  "postgres_database": "my_analytics",
-  "postgres_username": "myuser",
-  "postgres_password": "1234",
-  "postgres_schema": "mytapname"
-}
-```
-
-Run `target-postgres` against a [Singer](https://singer.io) stream.
-
+Create a JSON config file with the following parameters, and then run `target-postgres`
+against a [Singer](https://singer.io) stream:
+                                                                  
```

Comment:
Can we keep the example JSON? We can keep it simple with only the most common keys (host, database, username, password). We can also note it's an example.